config.toml

```diff
@@ -0,0 +1,85 @@
+[LOADED_NETWORK]
+loaded_network_directory = "Q:/Model Development/Simwrapper Validation/notebooks/roads/Data"
+
+# The dbf file name is LOAD[TOD].dbf by default
+loaded_network_time = ["AM", "MD", "PM", "EV", "EA"]
+
+# The needed CHAMP output columns
+loaded_network_col_names = ["A", "B", "AT", "FT", "V_1"]
+
+[OBSERVED]
+# The observed data file path
+obs_filepath = "Q:/Data/Observed/Validation/2019_old/observed_counts.csv"
+
+# The columns used for calculation, usually are 'A,B, and Counts'
+obs_usecols = ["A", "B", "Daily", "AM", "MD", "PM", "EV", "EA"]
+
+# The columns we want to include from the observed data, DONT CHANGE A, B!
+obs_extra_columns = ["A", "B", "Loc Type", "Source", "Source ID", "Location", "Dir"]
+
+[SCATTER_INPUT]
+# The time period we want to display in the scatter plot

```

Comment:
I'm confused about how the variables in this section are used.  They don't appear directly relevant to the scatter plots.

Reply:
This section does not need to change if we want the scatter plot to show daily data and classified by Loc Type like sf count, sf screenline. etc. But Chun Ho said I should set those values as variables not default values in the functions to cope with possible changes

Reply:
Ok, thanks for the explanation